pandas/io/stata.py

```diff
@@ -47,6 +58,8 @@
 
 from pandas.io.common import get_filepath_or_buffer, stringify_path
 
+BytesOrString = TypeVar("BytesOrString", str, bytes)
```

Comment:
although the alias name may be more explicit here, I _think_ it is common to just use AnyStr from typing, see https://docs.python.org/3/library/typing.html#typing.AnyStr

Reply:
Thanks.